Processes/Processes.py

```
@@ -82,7 +83,7 @@ def setup(self):
         self.processBoxes[processState] = processBox
         self.processLabels[processState] = processLabel
 
-    self.maximumRunningProcessesSpinBox.connect("valueChanged(double)", self.onMaximumChanged)
+    self.maximumRunningProcessesSpinBox.valueChanged.connect(self.onMaximumChanged)
```

I'm curious if this is really a better pattern.  I guess it generates a runtime error rather than just silently returning False (which isn't checked for) so I guess it's safer.  Maybe we should change all the connect instances to this pattern.  

But how does it handle overloaded signals?

---

https://stackoverflow.com/questions/11173028/connecting-an-overloaded-pyqt-signal-using-new-style-syntax
However I didn't succeed to make it for finishedSlot.

---

Interesting - looks like in PyQt there's a decorator option, but probably not in PythonQt.  For consistency let's stick with the explicit string declaration of the intended signal.

At run time you get a message like this on stderr if you try to connect to a non-existent signal, and that should be enough for developers.

`PythonQt: QObject::connect() signal 'hoot' does not exist on ctkDoubleSpinBox`